.github/workflows/swagger.yml

```diff
@@ -0,0 +1,17 @@
+name: Publish Swagger Specs
+
+on:
+  push:
+    branches:
+      - master
+      - AM-2797_Publish_Swagger_Docs
+  workflow_dispatch:
```

Comment:
This should be removed before merger.

Reply:
removed

Reply:
Resolved in e98225bd1c69790afa414a0ddf58d3d65cb3cead